src/app/loadout-drawer/loadout-apply.ts

```diff
@@ -412,6 +412,7 @@ function doApplyLoadout(
                   state.itemStates[initialItem.index].state =
                     // If we're doing a bulk equip later, set to MovedPendingEquip
                     itemsToEquip.length > 1 &&
+                    // TODO: The actual item may have a different ID, so we erroneously consider those items equipped!
```

Comment:
This (and the TODO below) means DIM sometimes fails to equip a subclass! I managed to provoke that issue with a loadout with a migrated void 3.0 subclass but this will also affect received loadouts. I'm unfamiliar with loadout application though.

Reply:
File some issues for these and I'll look at them as I continue messing with loadouts logic.